source/features/forked-to.tsx

```diff
@@ -0,0 +1,41 @@
+import './more-dropdown.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+async function init(): Promise<void> {
+	const forkDialog = select<HTMLElement>('details-dialog[src*="/fork"]')!;
+	const forkFragmentUrl = forkDialog.getAttribute('src')!;
+
+	const forkFragment = select<HTMLElement>('include-fragment', forkDialog)!;
+	forkFragment.addEventListener('load', () => onFragmentLoaded(forkDialog));
+	forkFragment.setAttribute('src', forkFragmentUrl); // This will load the fork fragment.
```

Comment:
This is not the suggested solution, I don’t want to load the fork fragment on every page of every repo I visit

Reply:
The fork fragment already starts loading once you *hover* over the fork button, therefor I thought GitHub wouldn't mind this **one** extra call to the fragment. It's **not** an API call. And it's done **async**.

Reply:
Yeah but I don’t hover the fork button and the suggested solution works best for repos that you didn’t fork: which is the vast majority of them.